### PR TITLE
Some changes to help building ads.

### PR DESCRIPTION
**Pull request by bobcassels**
- Add a startTime to the context given to ads.
   - Add note about this to ads/README.md
 - Move getAdCid to a separate function, rather than a method of AmpAd.
 - Fix a typo.

### CODE REVIEW

File: src/ad-cid.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import {userNotificationManagerFor} from './user-notification';
+
+
+/**
+ * @param {BaseElement} adElement
+ * @return {!Promise<string|undefined>} A promise for a CID or undefined if
+ *     - the ad network does not request one or
+ *     - `amp-analytics` which provides the CID service was not installed.
+ */
+export function getAdCid(adElement) {
```
Comment (cramforce) [inline src/ad-cid.js]: Would you mind moving some of the tests for the moved method into a dedicated test file? Thanks!

Comment (bobcassels) [inline src/ad-cid.js]: I don't see any existing tests for this part. But I will create some tests. Let me know if you are aware of tests I should move.

Comment (bobcassels) [inline src/ad-cid.js]: Done.

**Comment by cramforce** [comment]
LGTM

**Pull request merged**